src/config.rs

```diff
@@ -166,6 +167,29 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
```

Comment:
Nah, specifying `shell` without `program` seems kinda silly, and we can't actually provide a reasonable default in that case.